src/analysisd/eventinfo.c

```diff
@@ -1107,10 +1107,12 @@ char* ParseRuleComment(Eventinfo *lf) {
             field = lf->id;
         } else if (strcmp(var, "url") == 0) {
             field = lf->url;
-        } else if (strcmp(var, "data") == 0 || strcmp(var, "extra_data") == 0) {
+        } else if (strcmp(var, "data") == 0) {
             field = lf->data;
-        } else if (strcmp(var, "status") == 0) {
-            field = lf->status;
```

Comment:
Why has this assignment disappeared? `status` is a static field (indicated in the [documentation](https://documentation.wazuh.com/3.9/user-manual/ruleset/ruleset-xml-syntax/rules.html?highlight=rule%20syntax#status)) and this change causes it to be taken as a dynamic field.

In addition, this can cause it not to be added to the JSON formed by the `_getRulesListJSON` function.